datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -144,6 +142,29 @@ pub use writer::plan_to_parquet;
 /// * User provided  [`ParquetAccessPlan`]s to skip row groups and/or pages
 ///   based on external information. See "Implementing External Indexes" below
 ///
+/// # Predicate Pushdown
```

Comment:
I tried to consolidate the description of what predicate pushdown is done in the ParquetExec